packages/next/build/webpack/loaders/next-babel-loader.ts

```diff
@@ -0,0 +1,195 @@
+import hash from 'string-hash'
+import * as babel from '@babel/core'
+import { loader } from 'webpack'
+import { join, basename } from 'path'
+import loaderUtils from 'loader-utils'
+import cache from './next-babel-loader/cache'
+import transform, { version as transformVersion } from './next-babel-loader/transform'
+
+// increment 'a' to invalidate cache
+const cacheKey = 'babel-cache-' + 'a' + '-'
```

Comment:
We should probably bump this to be safe? 🤔 
```suggestion
const cacheKey = 'babel-cache-' + 'b' + '-'
```